app.js

```diff
@@ -1,156 +1,182 @@
 "use strict";
 
-let userName = prompt("What is your name?");
-let points = 0;
-{
-  userName = userName.toUpperCase();
-  alert("Hello and welcome "+ userName + "!");
-  points++;
-  console.log(points);
-}
+function questions(answer) {
 
-let askAge = prompt("What is my age?");
-if (askAge == 31) {
-  alert("You guessed my age correctly!");
-  points += 2;
-  console.log(points);
-} else if (askAge > 31 && askAge <= 35) {
-  alert("I'm not that old yet, but you are close.");
-  points += 1;
-  console.log(points);
-} else if (askAge < 31 && askAge >= 25) {
-  alert("I am not that young, but you are close.");
-  points += 1;
-  console.log(points);
-} else {
-  alert("You are not even close.");
+  answer = answer.toLowerCase();
+  if(answer === "yes" || answer === "y") {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Additionally, there's no reason you can't still track score. Functions can access variables defined outside of the function if you want them to!